src/main/java/konstructs/api/messages/InteractTertiaryFilter.java

```diff
@@ -0,0 +1,59 @@
+package konstructs.api.messages;
+
+import akka.actor.ActorRef;
+
+/**
+ * This class represents a {@link Filter} for the user's {@link InteractTertiary tertiary} (middle mouse button
+ * or button <code>e</code>) interaction event. The tertiary interaction event's default behaviour is to
+ * update the world with the block returned by {@link InteractTertiary#getBlockAtPosition()}. Any updates
+ * made to this field will therefore be reflected into the world (health, id, block type...). Therefore and since
+ * an unchanged message means that no updates are made to the world the tertiary interaction event should be
+ * used by plugins that want the user to interact either with a held block or a block in the world
+ * (or both simultaneously).
+ * <p>
+ *     Please note that the {@link #drop(ActorRef)} and
+ *     {@link #dropWith(ActorRef, InteractTertiary)} will <b>not</b> update the world with the block returned by
+ *     {@link InteractTertiary#getBlockAtPosition()}. Also note that if {@link InteractTertiary#getPosition()} is null,
+ *     it can not be updated and any change to {@link InteractTertiary#getBlockAtPosition()} will be ignore
+ *     (since the user is not pointing at anything).
+ * </p>
+ * @see InteractTertiary
+ */
+public class InteractTertiaryFilter extends Filter<InteractTertiary>{
+
+    /**
+     * Creates an immutable instance of the Filter.
+     * <b>This is used by the server internally and should not be done by a plugin.</b>
+     * @param chain The chain of plugins, a.k.a. event queue
+     * @param message The contained message
+     */
+    public InteractTertiaryFilter(ActorRef[] chain, InteractTertiary message) {
+        super(chain, message);
+    }
+
+    @Override
+    protected Filter<InteractTertiary> nextFilter(ActorRef[] chain) {
+        return new InteractTertiaryFilter(chain, getMessage());
+    }
+
+    @Override
+    protected Filter<InteractTertiary> nextFilter(ActorRef[] chain, InteractTertiary message) {
+        return new InteractTertiaryFilter(chain, message);
+    }
+
+    @Override
+    public void drop(ActorRef sender) {
```

Comment:
(not specific for this function, but a general question about all `@Override` with no comments, like this one.)

Maybe add a short explanation and/or link the other `drop`? Easier for the user browsing the documentation, what do you think?